### PR TITLE
Payments v2

### PR DESCRIPTION
**Pull request by tadovas**
TODO list:
- [x] Generate bindings of v0.0.2 smart contracts
- [x] Deploy smart contracts on localnet
Screenshot 2019-05-30 at 17 05 04

~~- [ ] Integrate payments v2 into node codebase~~
~~- [ ] Get rid of payments v1 repo depedency (and code) in deployer and node codebase~~

### CODE REVIEW

File: bin/localnet/deployer/deployer.go [hunk 1]
```diff
@@ -52,17 +56,67 @@ func main() {
 	checkError("backend lookup", err)
 	<-synced
 
+	// we still need to deploy legacy contracts to blockchain to be backwards compatible with current usage
+	deployLegacyContracts(transactor, client)
+
+	deployPaymentsv2Contracts(transactor, client)
+}
+
+func deployPaymentsv2Contracts(transactor *bind.TransactOpts, client *ethclient.Client) {
```
Comment (Waldz) [inline bin/localnet/deployer/deployer.go]: We already write Golang jobs as magefile.go tasks. Would be nice to connect this one there

Comment (tadovas) [reply bin/localnet/deployer/deployer.go]: Some day after tomorrow ;)

File: bin/localnet/deployer/deployer.go [hunk 2]
```diff
@@ -25,8 +25,12 @@ import (
 	"os"
 	"time"
 
+	"github.com/ethereum/go-ethereum/common"
+
```
Comment (vkuznecovas) [inline bin/localnet/deployer/deployer.go]: redundant whitespace :octocat:

**Comment by zolia** [comment]
I think we can merge this one and do rest of the tasks separately. It shouldn't break codebase.

**Comment by tadovas** [comment]
@zolia good point. Fixing rebasing, resubmiting it now.

**Pull request merged**